lib/megaLogger.js

```diff
@@ -80,12 +78,26 @@
     };
 
     /**
+     * Static, converts enum to text/label
+     *
+     * @private
+     * @static
+     */
+    var _intToLevel = {
+        '40': 'ERROR',
+        '30': 'WARN',
+        '20': 'INFO',
+        '10': 'LOG',
+        '0': 'DEBUG'
+    };
```

Comment:
The main idea here was to have one simple place that you can add/remove additional log levels (e.g. in MegaLogger.LEVELS).
However, i agree that the resources required for this makes this a bad idea :)

TLDR; nice, i will merge this :)